demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2273,15 +2284,64 @@ export class DemoMeetingApp
     this.log(`session stopped from ${JSON.stringify(sessionStatus)}`);
     this.log(`resetting stats in WebRTCStatsCollector`);
     this.statsCollector.resetStats();
+
+    const returnToStart = () => {
+      // If you want to make this a repeatable SPA, change this function to
+      // ```
+      // this.switchToFlow('flow-authenticate');
+      // ```
+      // and fix some state (e.g., video buttons).
+      //
+      // For simplicity, let's just reload the page.
+      window.location.href = window.location.pathname;
+    };
+
+    /**
+     * This is approximately the inverse of the initialization method above.
+     * This work only needs to be done if you want to continue using the page; if
+     * your app navigates away or closes the tab when done, you can let the browser
+     * clean up.
+     */
+    const onLeftMeeting = async () => {
+      // Clean up the timers for this.
+      this.audioVideo.unsubscribeFromActiveSpeakerDetector(this.activeSpeakerHandler);
+
+      // Stop listening to attendee presence.
+      this.audioVideo.realtimeUnsubscribeToAttendeeIdPresence(this.attendeeIdPresenceHandler);
+
+      // Stop watching device changes in the UI.
+      this.audioVideo.removeDeviceChangeObserver(this);
+
+      // Drop the audio input and output.
+      await this.audioVideo.chooseAudioInputDevice(null);
```

Comment:
Don't we already do it here https://github.com/aws/amazon-chime-sdk-js/blob/master/src/task/CleanStoppedSessionTask.ts#L59 ?

Reply:
Possibly. I think I didn't have this line to start with, but it seemed odd to have the video one but not this…

Reply:
Do we need to release video input device here too?

Reply:
Oh, I just realized — we cannot rely on `CleanStoppedSessionTask` having run, in part because they might be disposing before the task gets to run, and in part because they might never have started the meeting in the first place!